src/app/Classes/Table.php

```diff
@@ -53,9 +57,22 @@ public function templatePath()
 
     private function builder()
     {
-        return new Builder(
-            $this->request,
-            $this->query()
-        );
+        // Call query before builder
+        // Sometimes we need declare template path in current request
+        $query = $this->query();
+
+        return new Builder($this->request, $query, $this->getTemplate($this->templatePath()));
+    }
+
+    /**
+     * Get template object.
+     *
+     * @param string $path - path to template.json
+     *
+     * @return \stdClass
+     */
+    protected function getTemplate($path)
```

Comment:
What's this for?

Reply:
Yeah, you are right. It's too much. Will fix it.